apps/blog/src/posts/data-insights-on-PWAs/data-insights.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+title: PWA Insights - A Look at PWABuilder Data
```

Comment:
Consider changing to "Only X% of PWAs implement service workers - A quick review of the PWA ecosystem"